packages/xcoobee-cookie-kit-react/src/CookieKitContainer.js

```diff
@@ -418,6 +419,15 @@ export default class CookieKitContainer extends React.PureComponent {
             />
           </React.Fragment>
         )}
+        <BridgeCommunicator
+          ref={(bridgeRef) => {
+            this.bridgeRef = bridgeRef;
+          }}
+          campaignReference={campaignReference}
+          onCookieOptionsLoad={cookieOptions => this.resolveConnectedCookieConsents(cookieOptions)}
```

Comment:
Since you are just passing the parameter to another function, can't we just do the following instead?
```suggestion
          onCookieOptionsLoad={this.resolveConnectedCookieConsents}
```
And not create a new function every time `render` is called?

Reply:
I think you may have missed this comment. I think it may have be hidden behind a collapsed section that GitHub added.